static/src/components/VacancyForm.js

```diff
@@ -0,0 +1,207 @@
+/* eslint-disable react/prop-types */
+import {
+    Button,
+    Container,
+    Grid,
+    InputLabel,
+    MenuItem,
+    Select,
+    Slider,
+    TextField,
+    Typography,
+} from "@material-ui/core";
+import React, { useState, useEffect, useRef } from "react";
+import Autocomplete from "@material-ui/lab/Autocomplete";
+import InputItemsList from "./InputItemsList";
+
+const marks = [
+    {
+        value: 10000,
+        label: "10000₽",
+    },
+    {
+        value: 300000,
+        label: "300000₽",
+    },
+];
+
+const vacancies = ["Специалист по закупкам", "Финансовый менеджер"];
+
+const vacancyRecommendations = {
+    "Специалист по закупкам": [
+        [
+            "опыт работы от 3 лет",
+            "свободное владение английским",
+            "готовность к командировкам",
+            "уверенный пользователь ПК",
+        ],
+        [
+            "планирование бюджета закупок",
+            "ценообразование",
+            "ведение переговоров с поставщиками",
+            "создание асортиментной матрицы",
+        ],
+    ],
+    "Финансовый менеджер": [
+        [
+            "опыт работы от 1 года",
+            "высшее образование",
+            "знание налогового и бухгалтерского учета",
+        ],
+        [
+            "финансовый анализ ключевых показателей деятельности",
+            "разработка бюджетной и управленческой системы учета",
+        ],
+    ],
+};
+
+const defaultVacancy = {
+    salary: 45000,
+    position: "",
+    requirements: [],
+    responsibilities: [],
+};
+
+export default function VacancyFrom(props) {
```

Comment:
просто совет - удобнее использовать деструктуризацию https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

```suggestion
export default function VacancyFrom({ vacancy, position }) {
```